examples/Select.disable.tsx

```diff
@@ -3,27 +3,35 @@
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
 import * as React from 'react';
-import { Select } from '@itwin/itwinui-react';
+import { Select, Label } from '@itwin/itwinui-react';
 
 export default () => {
+  const labelId = React.useId();
+
   return (
-    <Select<number>
-      disabled
-      options={[
-        {
-          value: 1,
-          label: 'Item #1',
-        },
-        {
-          value: 2,
-          label: 'Item #2',
-        },
-        {
-          value: 3,
-          label: 'Item #3',
-        },
-      ]}
-      placeholder={'Placeholder text'}
-    />
+    <div>
+      <Label id={labelId}>Disabled Select Label</Label>
```

Comment:
can we instead show `LabeledSelect` in the examples? it handles everything (generally, it's better if the user doesn't need to worry about aria)

Reply:
Ohh, right, changed `<Label>` + `<Select>` examples to use `<LabeledSelect>` https://github.com/iTwin/iTwinUI/pull/1479/commits/44d332b9744bfb97d0ebf36683dd43eaa46ac103

Reply:
while you're changing the examples, could you also remove the unnecessary generics? e.g. `<Select<number>>` -> `<Select>`

Reply:
Sounds good, removed (https://github.com/iTwin/iTwinUI/pull/1479/commits/9bb9de411d115b32d0e9ee19922ed35098a98def)